action.yml

```
@@ -48,6 +52,7 @@ runs:
       shell: bash
     
     - id: deploy-to-s3
+      if: inputs.skip-deployment == 'false'
```

I was wondering if this would work or if we need something like that:
```suggestion
      if: ${{ inputs.skip-deployment == 'false' }}
```

---

did you try this version in your project? if you confirm that it works I'm happy to merge it as it is

---

Yes - my project is using the `if` statement without the brackets. I'm happy to commit the suggestion as it stands, but am not in a position to test the modified conditional.

---

fair enough, let's merge it